ndcube/ndcube.py

```diff
@@ -85,18 +85,19 @@ def extra_coords(self):
         pass
 
     @abc.abstractmethod
-    def crop_by_coords(self, lower_left_corner, dimension_widths):
+    def crop_by_coords(self, min_coord_values, interval_widths):
         """
-        Crops an NDCube given a lower left corner and widths of region of interest.
+        Crops an NDCube given minimum values and interval widths along axes.
 
         Parameters
         ----------
-        lower_left_corner: `list` of `astropy.units.Quantity`
-            The lower left corner of the region of interest described in physical units
-            consistent with the NDCube's wcs object.  The length of the iterable must
-            equal the number of data dimensions and must have the same order as the data.
+        min_coord_values: `list` of `astropy.units.Quantity`
+            The minimum desired values along each relevant axis after cropping
+            described in physical units consistent with the NDCube's wcs object.
+            The length of the iterable must equal the number of data dimensions and must
+            have the same order as the data.
 
-        dimension_widths: iterable of `astropy.units.Quantity`
+        interval_widths: iterable of `astropy.units.Quantity`
```

Comment:
same here. Also it's iterable here and `list` above. 'iterable' is more accurate, maybe it's worth standardising on that?

Reply:
Good idea!  Done.